src/services/OswServiceTaskConsole.cpp

```diff
@@ -78,11 +91,57 @@ void OswServiceTaskConsole::runPrompt() {
     }
     // command convention: show only what was asked (preferably machine readable), only on error be verbose
     bool processed = true;
-    if (this->m_configuring) {
+    switch (this->m_mode) {
+    case Mode::Generic:
+#ifdef OSW_FEATURE_BLE_SERVER
+        if (this->m_inputBuffer == "ble") {
+            this->m_mode = Mode::Ble;
+        } else if (this->m_inputBuffer == "configure") {
+#else
+        if (this->m_inputBuffer == "configure") {
+#endif
+            this->m_mode = Mode::Configuration;
+        } else if (this->m_inputBuffer == "help") {
+            this->showHelp();
+#if defined(OSW_FEATURE_WIFI) || defined(OSW_FEATURE_BLE_SERVER)
+        } else if (this->m_inputBuffer == "hostname") {
+            serial->println(OswConfigAllKeys::hostname.get());
+#endif
+        } else if (this->m_inputBuffer == "lock") {
+            this->m_locked = true;
+#ifndef OSW_EMULATOR
+        } else if (this->m_inputBuffer == "reboot") {
+            // this does not work in the emulator as it is running under an own thread, of which the shutdown-exception is not captured - populating here and crashing
+            ESP.restart();
+#endif
+#if OSW_SERVICE_NOTIFIER == 1
+        } else if (this->m_inputBuffer.find("toast ") == 0 and this->m_inputBuffer.length() > 6) {
+            auto arg = this->m_inputBuffer.substr(6);
+            std::string toast;
+            for (auto i = 0; i < arg.length(); i++) {
```

Comment:
rename `i`,

Reply:
It is the index... What do you propose? I think using something like `argIdx` (this even sounds misleading), `argumentCharacterIndex` or `index` does not yield additional value, especially since this `for`-loop is so short...

Reply:
IMHO - Sometimes we write simple and similar names for simple variables within a short range, but long codes seem to confuse.